### DISCUSSION TITLE
Strange behaviour

### DISCUSSION THREAD

**Discussion by basim1971** [codefori/vscode-ibmi#2185] [Q&A] [answered]
Starting from some days i face strange behaviour in vcode with ibmi extension. 
1) outline . In all the rpgle extension program outline is not visible. It's appear the message No symbol found in document. If a open a sqlrpgle program outline is well done
2) toggle line comment seem not work anymore
3) Using mouse right clik in rpgle extension appear a menu not in line with the standard . This work in sqlrpgle extension

Do you have any idea how solve it ? 
Thanks 

basilio

**Comment by worksofliam** [comment]
@basim1971 I converted this to a discussion since you didn't provide ways for us to recreate any of these issues.

**Comment by basim1971** [comment, marked as answer]
All i find this old discussion  https://github.com/orgs/codefori/discussions/1178
I disable RRPG by SpineNetSystems extension and now works fine

Thanks
